pylib/cqlshlib/test/test_cqlsh_output.py

```diff
@@ -587,10 +584,7 @@ def test_prompt(self):
             c.send('use NONEXISTENTKEYSPACE;\n')
             outputlines = c.read_to_next_prompt().splitlines()
 
-            start_index = 0
-            if c.realtty:
-                self.assertEqual(outputlines[start_index], 'use NONEXISTENTKEYSPACE;')
```

Comment:
Why did we remove the assertion check for index 0?

Reply:
moved back.